package.json

```diff
@@ -53,7 +53,7 @@
     "@vue/ref-transform": "^3.2.21",
     "@vue/shared": "^3.2.21",
     "defu": "^5.0.0",
-    "htmlparser2": "^7.2.0",
+    "htmlparser2": "5.0.1",
```

Comment:
Wait what? Downgrading a dependency doesn't feel right 🤔 does it?
Maybe the issue is then in the dependency and not in this repo?
Maybe it's better to fix the problem in the root.

Reply:
@Shinigami92 When I fix the htmlparser2 bug, I'll update the dependency to latest.